packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -257,9 +260,17 @@ const EAPostsItem = ({classes, ...props}: EAPostsItemProps) => {
     </>
   );
 
+  // The nesting here gets a little messy: we need to add the extra `Link`
+  // around the title to make it right-clickable/cmd+clickable. However,
+  // clicking this adds a second history item when navigating to the post
+  // normally requiring the user to press back twice to get to where they
+  // started so we need to wrap that whole thing in an `InteractionWrapper`
+  // too.
   const TitleWrapper: FC = ({children}) => (
     <PostsItemTooltipWrapper post={post} placement={tooltipPlacement} As="span">
-      {children}
+      <InteractionWrapper className={classes.titleWrapper}>
```

Comment:
I would guess that it would make sense for `InteractionWrapper` to have an `inline` prop.

Reply:
Makes sense - I'll save it for a separate PR though because people keep complaining about this bug and I want to deploy ASAP